src/components/layout/desktop_footer.tsx

```diff
@@ -0,0 +1,85 @@
+import * as React from "react"
+import styled from "styled-components"
+import * as fonts from "../../assets/fonts"
+
+const asideWidth = 28
+const columnWidth = (100 - asideWidth) / 4
+
+const Footer = styled.footer`
+  ${fonts.secondary.style}
+  background-color: white;
+  padding: 20px 0;
+  .column, .column-aside {
```

Comment:
Does this prefix the class names in any way / is this recommended styled-components usage? I thought the idea was to style the individual elements.